### PR TITLE
Prj 773 switch to kotest. Dummy tests refactoring

### PR DESCRIPTION
**Pull request by Avvessalom**
The beginning of the switching to kotest. Rewriting dummy-tests.

- projector-agent-common
- projector-ij-injector
- projector-util-agent

### CODE REVIEW

File: projector-util-agent/build.gradle.kts [hunk 1]
```diff
@@ -35,8 +35,14 @@ kotlin {
...
+val kotestVersion: String by project
+
 dependencies {
   implementation(project(":projector-util-logging"))
 
-  testImplementation(kotlin("test"))
+  testImplementation("io.kotest:kotest-assertions-core:$kotestVersion")
+  testImplementation("io.kotest:kotest-runner-junit5:$kotestVersion")
+}
+tasks.withType<Test> {
```
Comment (ARTI1208) [inline projector-util-agent/build.gradle.kts:46]:
```suggestion

tasks.withType<Test> {
```

**Review by ARTI1208 — APPROVED** [review]
Looks much better then initial variant. You could also consider following less wordy variant, but both are quite good for me
```
class DummyTest : StringSpec(
  {
    "empty test for coverage building" {
      true.shouldBeTrue()
    }
  }
)
```

File: projector-util-agent/build.gradle.kts (hunk 1 above)
Comment (Avvessalom) [reply projector-util-agent/build.gradle.kts:46]: I remembered that we declare a `useJunitPlatform` in a `setupJacoco()`. This block is not needed, I will remove it.

File: projector-agent-common/src/test/kotlin/org/jetbrains/projector/agent/common/DummyTest.kt [hunk 2]
```diff
@@ -23,12 +23,13 @@
...
 
-class MiscTest {
-  @Test
-  fun `empty test for coverage building`() {
-    assertTrue(true)
+class DummyTest : StringSpec() {
+  init {
+    "empty test for coverage building" {
+      true.shouldBeTrue()
+    }
```
Comment (SerVB) [inline projector-agent-common/src/test/kotlin/org/jetbrains/projector/agent/common/DummyTest.kt:33]:
```suggestion
    "empty test for coverage building" {}
```

As discussed, can we just remove the assertion (here and in other places)?

**Review by SerVB — APPROVED** [review]

**Pull request merged**